popup.js

```diff
@@ -43,17 +44,6 @@ const setUidValue = () => {
   });
 };
 
```

Comment:
Was environment id missing in the demo today?

Reply:
No and it's not working in master too.